.prettierrc.js

```diff
@@ -1,3 +1,6 @@
 module.exports = {
-  ...require('gts/.prettierrc.json')
-}
+  trailingComma: "es5",
+  tabWidth: 2,
+  semi: true,
+  singleQuote: true,
+};
```

Comment:
We’re going to need to bike shed these at some point, haha. But now is not the time